docs/components-accordion.md

```diff
@@ -0,0 +1,84 @@
+# Components
+
+This theme ships some components (as twig macros) that hide the complexity of rendering the same elements over and over again with the correct HTML.
+
+See Tabler documentation at https://preview.tabler.io/accordion.html
+
+## Accordion
+
+### Parameters
+`accordion()` macro, waits for 2 parameters:
+
+| Parameter | Description                |   Type   | Default |
+|:---------:|:---------------------------|:--------:|:-------:|
+|   items   | Array of [Item](#Item)     | `array`  |  `[]`   |
+|  options  | [Options](#Options) object | `object` |  `{}`   |
+
+#### Item
+| Parameter | Description                        |   Type   |    Default     |
+|:---------:|------------------------------------|:--------:|:--------------:|
+|   title   | Url href for the step when clicked | `string` |      `#`       |
+|   body    | Title of the step                  | `string` | _empty string_ |
+|  options  | Customize the item (see below)     | `object` |      `{}`      |
+
+##### Item Options
+|    Parameter    | Description                  |   Type   |    Default     |
+|:---------------:|------------------------------|:--------:|:--------------:|
+| titleExtraClass | Add extra class to the title | `string` | _empty string_ |
+| bodyExtraClass  | Add extra class to the body  | `string` | _empty string_ |
+
+#### Options
+|  Parameter  | Description                              |   Type    |    Default     |
+|:-----------:|------------------------------------------|:---------:|:--------------:|
+|     id      | Set custom `id` to accordion             |  `mixed`  | `rand(number)` |                                        
+|     raw     | Use raw output for `title` and `body`    | `boolean` |     `true`     |                                        
+|    flush    | Remove padding (see Tabler docs)         | `boolean` |    `false`     | 
```

Comment:
I'm not OK with flush option description.
There's no information about that in Tabler, but can be found with Bootstrap https://getbootstrap.com/docs/5.0/components/accordion/#flush